util/types/src/core/hardfork.rs

```diff
@@ -209,6 +216,14 @@ define_methods!(
     "RFC PR 0237"
 );
 
+define_methods!(
+    rfc_pr_0234,
+    p2p_network_switch,
+    is_p2p_network_switch_enabled,
```

Comment:
I think we could use the same switch, since they have to equal to each other.

Reply:
I agree. https://github.com/nervosnetwork/ckb/pull/2765#discussion_r661295725

Reply:
Change 00237 to 234 + 237, so it will share the same value